paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/settings/PlaygroundConfigurationData.kt

```diff
@@ -22,14 +22,31 @@ data class PlaygroundConfigurationData(
         Embedded,
 
         @SerialName("CustomerSheet")
-        CustomerSheet;
+        CustomerSheet,
+
+        @SerialName("FlowControllerWithSpt")
+        FlowControllerWithSpt;
 
         fun isPaymentFlow(): Boolean {
-            return this in setOf(PaymentSheet, FlowController, Embedded)
+            return this in paymentFlows()
+        }
+
+        fun isSptFlow(): Boolean {
+            return this in sptFlows()
         }
 
         fun isCustomerFlow(): Boolean {
             return this == CustomerSheet
         }
+
+        companion object {
+            fun paymentFlows(): Set<IntegrationType> {
+                return setOf(PaymentSheet, FlowController, Embedded)
+            }
+
+            fun sptFlows(): Set<IntegrationType> {
+                return setOf(FlowControllerWithSpt)
+            }
```

Comment:
Their usage confused me a little in the settings definitions. Let's follow up to more clearly name these.